Clippit/PowerPoint/FluentPresentationBuilder.cs

```diff
@@ -672,8 +672,6 @@ private void CopyRelatedPartsForContentParts(OpenXmlPart oldContentPart, OpenXml
             foreach (var chartReference in newContent.DescendantsAndSelf(C.chart))
             {
                 var relId = (string)chartReference.Attribute(R.id);
-                if (newContentPart.HasRelationship(relId))
-                    continue;
```

Comment:
Here we enumerate all `chartReference`'s that we need to copy and copy ones that already exist in the new part. I believe that this if statement assumes that `ChartPart` is already copied.

With this change, you copy everything again, overwrite `relId`, and potentially leave unused (unnecessary) parts inside the new presentation